account_due_list_payment_mode_edit/readme/CONTRIBUTORS.md

```diff
@@ -0,0 +1,2 @@
+* Iria Alonso \<iria.alonso@solvos.es\>
```

Comment:
Original author (first commit author)is missing (Omar Castiñeira - Comunitea S.L. <omar@comunitea.com>)